service/api_spawn.go

```diff
@@ -162,6 +162,7 @@ func getDockerfile(w http.ResponseWriter, r *http.Request) {
 		"ARG BINARY_NAME",
 		"ADD ${URL}/clients/${EXECUTABLE_SUB_PATH} /root/",
 		"RUN chmod +x /root/${BINARY_NAME}",
+		"COPY /root/.ssh /root/.ssh",
```

Comment:
this assumes things about the parent distro (e.g. that /root is readable, that there's a key there, etc.) all of which is true for arch linux, but is probably not true for any other distro. 

if this is an attempt to fix the thing that I think it's trying to fix, can you just specify token auth for git.get_projects?

Reply:
I mentioned it in the pr, but I didn't go into enough detail. Since this dockerfile already makes assumptions about the file structure on arch, we'll need to refactor it at some point (which could be now if you'd like). This is trying to get our test that git clones over ssh to work, which won't be fixed by that since it's not using git.get_project

Reply:
I was suggesting that we avoid doing git clones over ssh all together. (which git.get_project is capable.)